src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -515,7 +515,7 @@ private static function initPlaceholderFormatters(): array
                 $completeBars = $bar->getBarOffset();
                 $display = str_repeat($bar->getBarCharacter(), $completeBars);
                 if ($completeBars < $bar->getBarWidth()) {
-                    $emptyBars = $bar->getBarWidth() - $completeBars - Helper::length(Helper::removeDecoration($output->getFormatter(), $bar->getProgressCharacter()));
+                    $emptyBars = max(0, $bar->getBarWidth() - $completeBars - Helper::length(Helper::removeDecoration($output->getFormatter(), $bar->getProgressCharacter())));
```

Comment:
IMHO this solution is not workling correctly, because now the progress bar will be longer ... using the example of your test, just to show the difference:
```
  7/10 [===================>>>>-----]  70%
  8/10 [======================>>>>--]  80%
  9/10 [=========================>>>>]  90%
```

Reply:
Thank you for pointing this out, I'll try to find a solution!